.github/workflows/torch-tests.yml

```diff
@@ -0,0 +1,59 @@
+name: TorchScript tests
```

Comment:
We will also need in the future a workflow compiled with CUDA, right?

Reply:
Only if we can get a CI maching able to run CUDA code. I wanted to experiment with self-hosted github action runner, and use the imx server for this, it should be doable (we only need a demon on the head node that stays conected to github and answers requests).